test_aodndata/aodn_wave_dm/test_aodn_wave_dm.py

```diff
@@ -96,6 +96,24 @@ def test_dest_path_aodn_wave_dm(self):
                              'Southern_Ocean',
                              os.path.basename(good_nc)))
 
+        good_nc = os.path.join(TEST_ROOT, 'IMOS_NTP-WAVE_TW_20200113T215850Z_TOR01_WAVERIDER_FV01_timeseries_END-20200319T015105Z.nc')
```

Comment:
FYI: This file name doesn't match the IMOS file naming convention:
`IMOS_<Facility-Code>_<Data-Code>_<Start-date>_<Platform-Code>_FV<File-Version>_<Product-Type>_<End-date>_<Creation-date>_<PARTX>.nc`

Between `<Start-date>` and `FV01`, there should only be a single field `<Platform-Code>`. You have two fields there `TOR01` and `WAVERIDER`. Which is the platform code? If you want to keep them both there, you should put a hyphen between them, instead of an underscore.

Reply:
Alternatively, (e.g. if this file were part of the moorings facility) you might think of "WAVERIDER" as the _instrument_, and put it in the `<Product-Type>` field:
`IMOS_NTP-WAVE_TW_20200113T215850Z_TOR01_FV01_WAVERIDER-timeseries_END-20200319T015105Z.nc`

Reply:
Yes, @mhidas the filename does not follow the IMOS convention, but it is consistent with the format adopted for the files of the National wave archive which this dataset will be part of.